features/manage-position/Withdraw.tsx

```diff
@@ -246,19 +293,19 @@ const Deposit = () => {
           <ul style={{ fontSize: 16 }}>
             <li>
               <strong>"Fast" withdrawal: </strong>Instantly withdraw collateral
```

Comment:
this is defs cleaner but do we really want to use acronyms like this?

Reply:
I thiink so because on the bottom part we display the CR's with acronyms:
![Screen Shot 2020-07-24 at 08 41 35](https://user-images.githubusercontent.com/9457025/88392139-7d9ee380-cd89-11ea-9355-cb44b5a9eec4.png)